betty/project/extension/raspberry_mint/webpack/css/overlay.scss

```diff
@@ -0,0 +1,5 @@
+@import "bootstrap/scss/mixins/breakpoints";
+
+.overlay-background-contrast {
```

Comment:
Extract this into a separate PR and apply it to the trees' full screen button as well